fusor-ember-cli/tests/unit/serializers/node-test.js

```diff
@@ -1,15 +1,15 @@
 import { moduleForModel, test } from 'ember-qunit';
 
-moduleForModel('foreman-task', 'Unit | Serializer | foreman task', {
+moduleForModel('node', 'Unit | Serializer | node', {
```

Comment:
I actually deleted the serializer test for `foreman-task` (since the serializer was also deleted as it just uses the application serializer), and added node serailizer test, but git made it look like it was a rename. Not sure why.